proto/src/determined/experiment/v1/experiment.proto

```diff
@@ -68,6 +68,8 @@ message Experiment {
   string username = 10;
   // The resource pool the experiment was created in
   string resource_pool = 11;
+  // The type of searcher for the experiment
+  string searcher_type = 12;
```

Comment:
this is also used in the singular version of the API `experiments/:id` which uses another SQL file than the one you modified. could you update that one as well? 
we could probably refactor to avoid the need for almost duplicated query (sql files) to get ExperimentV1 object.

Reply:
Fixed